src/index.js

```diff
@@ -1,8 +1,9 @@
 import window from 'global/window';
 import console from 'global/console';
+import {get} from '@polymer/polymer/lib/utils/path';
```

Comment:
```js
import {get} from '../../@polymer/polymer/lib/utils/path';
```

Reply:
right! I think it will be a bit longer to fix all the repo (I have first to understand it 😄 ) so it will take some commits to have everything working fine... Vaadin combo box hasnt yet a v3 version so I will work on that first maybe

Reply:
Because polymer-redux is a library and not part of the application need to make sure the imports following other polymer libraries.

Reply:
unfortunately it seems that the "suggested" way of import from the Polymer team does not always work, I cant get the tests to run and the code to be built at the sime time, relatives paths are always a mess.

Reply:
With Webpack builds this problem is solved, so I'm still using my fork on my company projects. Can't get how to make things mergeable with the actual repo structure 😞 (HTML Imports stuff)